db/model/MultiDimDataPage.ts

```diff
@@ -1,10 +1,25 @@
-import { KnexReadonlyTransaction } from "../db.js"
+import { KnexReadonlyTransaction, KnexReadWriteTransaction } from "../db.js"
 import {
     MultiDimDataPagesTableName,
+    DbInsertMultiDimDataPage,
     DbPlainMultiDimDataPage,
     DbEnrichedMultiDimDataPage,
 } from "@ourworldindata/types"
 
+/** Returns ID of the upserted row or zero, if there are no updates. */
```

Comment:
I don't understand in which case this would return zero. Can you elaborate?

Reply:
I reworded the doc. Is it clear now?

Reply:
Yes it is - but this behaviour isn't very obvious to me from the docs. Have you verified that this really returns 0 if there is no difference in the upserted data? I'm very happy to run with this if you did but I'd like to avoid a codepath that we add that, for whatever reasons, in practice doesn't happen like this.

Reply:
I did test it. That's how I found about it. I just double-checked and yep, that's how it works.

Reply:
Perfect, thanks! Good to go then!